lib/lily-pad.js

```diff
@@ -6,8 +6,8 @@ function LilyPad(x, y){
 }
 
 LilyPad.prototype.draw = function(ctx){
-  ctx.fillRect(this.x, this.y, this.width, this.height);
   ctx.fillStyle = "black";
-};
+  ctx.fillRect(this.x, this.y, this.width, this.height);
+ };
```

Comment:
I'm glad you found this. I was banging my head against a wall trying to figure that out.